src/services/ParticipantProjectSpeciesService.ts

```diff
@@ -0,0 +1,118 @@
+import { components } from 'src/api/types/generated-schema';
+
+import { Response } from './HttpService';
+
+export type ProjectSpecies = {
+  deliverableId: number;
+  id: number;
+  projectId: number;
+  speciesId: number;
```

Comment:
You can add `rationale`, `status`, `feedback` to this type.

Reply:
You got it. Are `rationale` and `feedback` both `string`s and is `status` an enum?

Reply:
You can do all strings for now, enum will come from BE when it is done